setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name = "a10-neutron-lbaas",
-    version = "1.3.3",
+    version = "1.3.4",
     packages = find_packages(),
 
     author = "A10 Networks",
```

Comment:
The acos_client min version needs to be tweaked here, and then this is good to go.